Orthtree/doc/Orthtree/Concepts/CollectionPartitioningOrthtreeTraits.h

```diff
@@ -0,0 +1,62 @@
+/*!
+  \ingroup PkgOrthtreeConcepts
+  \cgalConcept
+
+  Refinement of the `OrthtreeTraitsWithData` concept, adding requirements for the
+  traits class of a `CGAL::Orthtree` in order to supports nearest-neighbor searching.
+
+  Nearest neighbor searches expect a tree where `Node_data` is a model of `ForwardRange`.
+  The leaf nodes of the tree represent an exclusive partition of the elements contained in the tree.
+  This means that no element should be contained by more than one node.
+
+  \cgalRefines{OrthtreeTraitsWithData}
+
+  \cgalHasModelsBegin
+  \cgalHasModels{CGAL::Orthtree_traits_point<GeomTraits, PointRange, PointMap, DimensionTag>}
+  \cgalHasModelsEnd
+*/
+class CollectionPartitioningOrthtreeTraits {
+public:
+
+
+  /// \name Types
+  /// @{
+
+  /*!
+   * Sphere type used for the shrinking-sphere approach for neighbor queries
+   */
+  using Sphere_d = unspecified_type;
+
+  /*!
+   * \brief The data type contained by each node; must be a model of `ForwardRange`.
+   */
+  using Node_data = unspecified_type;
+
+  /*!
+   * \brief An element of the `Node_data` list-like type.
+   *
+   * Must be constructible from the value type of a `Node_data` iterator.
```

Comment:
so it's simply Node_data, no?

Reply:
No, Node_data is a range and Node_data_element is boost::range_value<Node_data>.

Reply:
so maybe directly write Node_data::iterator?